packages/terra-switch/README.md

```diff
@@ -0,0 +1,46 @@
+# Terra Switch
+
+
+[![NPM version](https://badgen.net/npm/v/terra-switch)](https://www.npmjs.org/package/terra-switch)
+[![Build Status](https://badgen.net/travis/cerner/terra-switch)](https://travis-ci.com/cerner/terra-core)
+
+The terra-switch component are like their real-world counterparts, are used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
+
+Switches are used to activate settings immediately, in real time. 
+Switches never require users to press a button to apply their settings. 
+A switch is a set of binary options and always has one of the options selected by default.
+
+Switch should be used for only one of these set of binary options:
+• On / Off
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-switch/docs)
```

Comment:
Since there will be no docs folder you can change this to point to the engineering site.

Reply:
updated